articles/container-service/container-service-kubernetes-walkthrough.md

```diff
@@ -22,118 +22,161 @@ ms.author: anhowe
+
+### Remote sessions inside your containers
+Kubernetes allows you to run commands in a remote Docker container running in your cluster.
+
+```console
+# Get the name of your nginx pod
+kubectl get pods
+```
+
+Using your pod name, you can run a remote command on your pod.  For example:
+```console
+kubectl exec nginx-701339712-retbj date
+```
+
+You can also get a fully interactive session using the `-it` flags:
+
+```console
+kubectl exec nginx-701339712-retbj -it bash
+```
+
+![Image of curl to podIP](media/container-service-kubernetes-walkthrough/kubernetes-remote.png)
+
+
+## Details
+### Installing the kubectl configuration file
+When you ran `az acs kubernetes get-credentials`, the kube config file for remote access was stored under the home directory ~/.kube/config.
+
+If you ever need to download it directly, you can use `ssh` on Linux or OS X, or `Putty` on windows:
+
+#### Windows
+To use pscp from [putty](http://www.chiark.greenend.org.uk/~sgtatham/putty/download.html).  Ensure you have your certificate exposed through [pageant](https://github.com/Azure/acs-engine/blob/master/docs/ssh.md#key-management-and-agent-forwarding-with-windows-pageant):
   ```
   # MASTERFQDN is obtained in step1
   pscp azureuser@MASTERFQDN:.kube/config .
   SET KUBECONFIG=%CD%\config
   kubectl get nodes
   ```
-  * OS X or Linux:
+
+#### OS X or Linux:
   ```
   # MASTERFQDN is obtained in step1
   scp azureuser@MASTERFQDN:.kube/config .
   export KUBECONFIG=`pwd`/config
   kubectl get nodes
   ```
-10. The next step is to show you how to remotely run commands in a remote Docker container:
-  1. Run `kubectl get pods` to show the name of your nginx pod
-  2. using your pod name, you can run a remote command on your pod.  eg. `kubectl exec nginx-701339712-retbj date`
-  3. try running a remote bash session. eg. `kubectl exec nginx-701339712-retbj -it bash`.  The following screen shot shows these commands:
-
-  ![Image of curl to podIP](media/container-service-kubernetes-walkthrough/kubernetes-remote.png)
-
-11. The final step of this tutorial is to show you the dashboard:
-  1. run `kubectl proxy` to directly connect to the proxy
-  2. in your browser browse to the [dashboard](http://127.0.0.1:8001/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard/#/workload?namespace=_all)
-  3. browse around and explore your pods and services.
-  ![Image of Kubernetes dashboard](media/container-service-kubernetes-walkthrough/kubernetes-dashboard.png)
-
 # Learning More
 
-Here are recommended links to learn more about Kubernetes:
+## Azure Container Service
 
-1. [Azure Kubernetes documentation](https://azure.microsoft.com/en-us/documentation/services/container-service/)
+1. [Azure Container Service documentation](https://azure.microsoft.com/en-us/documentation/services/container-service/)
+2. [Azure Container Service Open Source Engine](https://github.com/azure/acs-engine)
 
 ## Kubernetes Community Documentation
```

Comment:
Can you update the community docs with these great links you sent the other day:

https://katacoda.com/embed/kubernetes-bootcamp/1/

and then maybe a section on Kubernetes + Production Talks:
TicketMaster:
https://www.youtube.com/watch?v=PSPNg5AU_II

Comcast:
https://www.youtube.com/watch?v=lmeFkH-rHII

Buffer:
https://www.youtube.com/watch?v=EC_ZRLsw58M

Concur:
https://www.youtube.com/watch?v=eQ9R8prQUHU